ffi/build.py

```diff
@@ -100,7 +101,15 @@ def main_windows():
     # Run configuration step
     try_cmake(here_dir, build_dir, *generator)
     subprocess.check_call(['cmake', '--build', build_dir, '--config', config])
-    shutil.copy(os.path.join(build_dir, config, 'llvmlite.dll'), target_dir)
+    try:
+        shutil.copy(os.path.join(build_dir, config, 'llvmlite.dll'), target_dir)
+    except shutil.SameFileError:
```

Comment:
Why would this happen? And if it would, why just swallow / ignore the exception?

Reply:
Yeah, you're right. Need to remove from part from the original PR as well